scripts/spack/packages/serac/package.py

```diff
@@ -111,7 +111,7 @@ class Serac(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("netcdf-c@4.7.4")
 
-    depends_on("hypre@2.26.0~superlu-dist+mpi")
+    depends_on("hypre~superlu-dist+mpi")
```

Comment:
Should this maybe be a lower bound? I assume at some point a version won't work...

Reply:
agreed, i changed it